packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin-conf

```diff
@@ -1,10 +1,12 @@
 [required-extensions]
 kubelet-device-plugins = "v1"
+container-runtime = "v1"
```

Comment:
Why do you need the container runtime settings here? They are not used anywhere in this file.